scripts/ci/detect_bookrunner_failures.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Is this needed?

Reply:
No, will purge.

Reply:
00ca7fd5a46d3733dbf09d49fa8c556f38af1e18